tests/testthat/test-drive_list.R

```diff
@@ -8,97 +8,133 @@ context("List files")
 # (pesky_files <- drive_list(pattern = "foo|bar|baz|yo"))
 # pesky_files$id %>% purrr::map(drive_file) %>% purrr::map(drive_delete)
 
+nm_ <- function(x) paste0(x, "-TEST-drive-list")
+
+run <- FALSE
+clean <- FALSE
+if (run) {
+  ## make sure directory is clean
+  if (clean) {
+    del_ids <- drive_list(pattern = paste(c(nm_("foo"),
+                                            nm_("foo2"),
+                                            nm_("bar"),
+                                            nm_("baz"),
+                                            nm_("yo")),
```

Comment:
`nm_()` is vectorized, so I think this can be `paste(nm_(c("foo", "foo2", "bar", "baz", "yo")), collapse = "|")`.